### PR TITLE
K8SPG-355 create cluster wide doc

### PR DESCRIPTION
**Pull request by fiowro**

### CODE REVIEW

File: docs/cluster-wide.md [hunk 1]
```diff
@@ -0,0 +1,163 @@
...
+2. Deploy the bundle. It will create the `pg-operator` namespace, and deploy the
+    Operator into it.
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply -f deploy/cw-bundle.yaml –server-side
+    ```
+
+    !!! note
+
+        If you want to change the namespace, please edit
```
Comment (hors) [inline docs/cluster-wide.md:106]: I discussed it with Ege yesterday, and Ege will improve it in the same way as we have for all other operators. Please wait for changes.

File: docs/cluster-wide.md [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+
+    Right now the operator deployed in cluster-wide mode will monitor all
+    namespaces in the cluster, either already existing or newly created ones.
+
+3. Switch to the namespace you have chosen for the cluster (don't forget to
+    create it if needed). let's call it `percona-db-1` for example:
+    
+    ``` {.bash data-prompt="$" }
+    $ kubectl create namespace percona-db-1
+    $ kubectl config set-context $(kubectl config current-context) --namespace=percona-db-1
```
Comment (hors) [inline docs/cluster-wide.md:148]: You should not do it you can just specify the namespace in line 154

File: docs/cluster-wide.md [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+    ```
+    
+4.  Deploy the cluster in the namespace of your choice:
+
+    ``` {.bash data-prompt="$" }
+    $ kubectl apply -f deploy/cr.yaml
```
Comment (hors) [inline docs/cluster-wide.md:154]:
```suggestion
    $ kubectl apply -f deploy/cr.yaml -n percona-db-1
```

**Review by hors — CHANGES_REQUESTED** [review]

File: docs/gke.md [hunk 4]
```diff
@@ -128,22 +129,18 @@ $ kubectl create clusterrolebinding cluster-admin-binding --clusterrole cluster-
         perconapgcluster.pg.percona.com/cluster1 created
         ```
 
-    Creation process will take some time. The process is over when the Operator
-    and PostgreSQL Pods have reached their Running status:
+    Creation process will take some time. The process is over when both
+    Operator and replica set Pods have reached their Running status:
```
Comment (egegunes) [inline docs/gke.md:133]:
```suggestion
    Operator and PostgreSQL Pods have reached their Running status:
```

**Review by egegunes — APPROVED** [review]

**Review by hors — APPROVED** [review]

**Pull request merged**